Watchman.Discord/Areas/Initialization/Controllers/InitializationController.cs

```diff
@@ -20,13 +20,15 @@ public class InitializationController : IController
         private readonly ICommandBus _commandBus;
         private readonly MuteRoleInitService _muteRoleInitService;
         private readonly UsersRolesService _usersRolesService;
+        private readonly ScanMessagesHistoryService _scanMessagesHistoryService;
 
-        public InitializationController(IQueryBus queryBus, ICommandBus commandBus, MuteRoleInitService muteRoleInitService, UsersRolesService usersRolesService, UsersService usersService)
+        public InitializationController(IQueryBus queryBus, ICommandBus commandBus, MuteRoleInitService muteRoleInitService, UsersRolesService usersRolesService, ScanMessagesHistoryService scanMessagesHistoryService)
```

Comment:
```suggestion
        public InitializationController(IQueryBus queryBus, ICommandBus commandBus, MuteRoleInitService muteRoleInitService, UsersRolesService usersRolesService, MessagesHistoryService messagesHistoryService)
```
tak, wiem że wtedy dwie klasy będą mieć nazwe MessagesHistoryService 
myśle że to powinien być jeden serwis 

no chyba że z jakiegoś powodu warto je rozdzielić 

albo ten nazwać jako np MessagesHistoryScanner
albo ServerScanningService 
albo coś w tym stylu